### PR TITLE
Add state utilization apis

### PR DESCRIPTION
**Pull request by liayoo**
- Added `ain_getStateUsage` json-rpc API
- Added `get_state_usage` client API

Related: https://github.com/ainblockchain/ain-blockchain/issues/498

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -129,6 +130,18 @@ app.get('/get_state_info', (req, res, next) => {
     .end();
 });
 
+/**
+ * Returns the state usage of the given app.
+ */
+ app.get('/get_state_usage', (req, res, next) => {
+  const result = req.query.app_name
+      ? node.db.getStateUsageAtPath(`${PredefinedDbPaths.APPS}/${req.query.app_name}`) : null;
```
Comment (platfowner) [inline client/index.js:138]: what about introducing a function in node/index.js to use both here and in ain_getStateUsage() for future consistency?

minor: could you move '?' operator to above line?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: client/index.js (hunk 1 above)
Comment (liayoo) [reply client/index.js]: Great idea! Done!

**Comment by liayoo** [comment]
Thanks for the review! Now merging~

**Pull request merged**